### DISCUSSION TITLE
Spinner does not work

### DISCUSSION THREAD

**Discussion by BTityo** [bootstrap-vue/bootstrap-vue#6203] [Q&A] [answered]
Hi!
I would like to use the <b-overlay> with spinner. The overlay works for me, but spinner doesn't. I attached a picture from code.
![Spinner](https://user-images.githubusercontent.com/40918916/102208880-44017d00-3ed0-11eb-876e-6654ed786d11.png)

packages:
  "dependencies": {
    "axios": "^0.21.0",
    "bootstrap": "4.3.1",
    "bootstrap-vue": "^2.21.0",
    "chart.js": "^2.9.3",
    "d3": "^5.7.0",
    "datamaps": "^0.5.9",
    "date-fns": "^1.30.1",
    "dropzone": "^5.5.1",
    "element-ui": "2.4.11",
    "es6-promise": "^4.1.1",
    "flatpickr": "^4.5.7",
    "fuse.js": "^3.2.0",
    "google-maps": "^3.2.1",
    "nouislider": "^12.1.0",
    "perfect-scrollbar": "^1.3.0",
    "quill": "^1.3.6",
    "sweetalert2": "^9.5.4",
    "vee-validate": "^3.2.1",
    "vue": "^2.6.11",
    "vue-chartjs": "^3.5.0",
    "vue-clipboard2": "^0.3.0",
    "vue-flatpickr-component": "^8.1.2",
    "vue-google-login": "^2.0.5",
    "vue-router": "^3.0.6",
    "vue2-transitions": "^0.2.3"
  },

Thanks in advance!

**Comment by Hiws** [comment, marked as answer]
My guess would be it's a CSS issue.

Are there any styles being applied to the spinner elements if you inspect them in your browser?
How are you importing Bootstrap CSS?

**Reply by BTityo** [reply]
Ohhh, thank you!
I typed wrong css path :)

Thanks, now it's working correctly!
